pkg/scalers/kafka_scaler.go

```diff
@@ -592,7 +592,10 @@ func (s *kafkaScaler) getTotalLag() (int64, int64, error) {
 
 	for topic, partitionsOffsets := range producerOffsets {
 		for partition := range partitionsOffsets {
-			lag, lagWithPersistent, _ := s.getLagForPartition(topic, partition, consumerOffsets, producerOffsets)
+			lag, lagWithPersistent, err := s.getLagForPartition(topic, partition, consumerOffsets, producerOffsets)
+			if err != nil {
+				return 0, 0, err
```

Comment:
might be benefitical to pring error log message here as well, with the info about the problematci partition.

Reply:
could you please elaborate? Since the error log message is already printed here
https://github.com/kedacore/keda/blob/415b48335be73ed5887ca3d756a83d33a98df619/pkg/scalers/kafka_scaler.go#L447-L449
We just not ignoring it anymore. Or am I misunderstanding something?

Reply:
You are right, I missed this one. Please disregard :)